Engine/Graphics/DecalBuilder.h

```diff
@@ -38,130 +34,101 @@ struct Bloodsplat {
     unsigned char r;
     unsigned char g;
     unsigned char b;
-    char field_1B;
-    int field_1C;  // flags
-    unsigned long long field_20;  // fade time
-    // int field_24;
+    int blood_flags;  // &1 for fade
+    uint64_t fade_timer;
 };
 #pragma pack(pop)
 
-/*  159 */
+// store for all the bloodsplats to be applied
 #pragma pack(push, 1)
```

Comment:
i think we can remove pragma pack push and pop (after the structure definition) at this point here